src/scripts/costing/README.md

```diff
@@ -0,0 +1,112 @@
+# Cost Estimation Utility Functions (`cost_estimation.py`)
```

Comment:
This is beautiful!!

The only thing I wonder whether it would be helpful to add would be a vignette of actual use.

e.g. Just to show that we start with a Path object to a simulation and then we can use, in turn, each of these functions.

It's visible from the scripts that are done, but these obviously have a lot of other things going on that might obscure the basics.

This vignette _could_ be in the form of a test whereby we run through all the commands using a very small simulation that is stored in tests. This is what we're doing with `tests/resources/dummy_simulation_run`, where that run is the smallest run sufficient for the functions to all work on it. 

Having it tested, would ensure that the functionality keeps working despite any future changes.